### ISSUE TITLE
Can't disenchant blacksmithing items

### ISSUE THREAD

**Issue by LilPoira** [SunwellWoW/Sunwell-TBC-Bugtracker#2479]
Description: 
Cannot disenchant green-level items created by a blacksmith.

How it works: 
Items say "Disenchanting requires Enchanting (65525)"

How it should work: 
You should be able to disenchant these crafted items as normal.

Source (you should point out proofs of your report, please give us some source): 
Easily tested in-game

**Comment by tihiniko** [comment]
Yes please fix this. This is a silly bug that shouldn't have been there at release

**Comment by Taolos** [comment]
I was about to write a ticket but you did it already. Plz fix it, this is really really annoying...

**Comment by Humangnome** [comment]
I am also experiencing this issue with Mail items crafted by other users, requires Enchanting 65535

**Comment by akrom23** [comment]
#2902
